packages/aws-cdk-lib/aws-ec2/lib/launch-template.ts

```diff
@@ -146,6 +146,53 @@ export enum SpotRequestType {
   PERSISTENT = 'persistent',
 }
 
+/**
+ * Capacity Reservation preferences
+ */
+export enum CapacityReservationPreference {
+  /** Launch instances into any available Capacity Reservation */
+  OPEN = 'open',
+  /** Do not launch instances into Capacity Reservations */
+  NONE = 'none',
+  /** Only launch instances into Capacity Reservations */
+  CAPACITY_RESERVATIONS_ONLY = 'capacity-reservations-only',
+}
+
+/**
+ * Target Capacity Reservation specification
+ */
+export interface LaunchTemplateCapacityReservationTarget {
+  /**
+   * The ID of the Capacity Reservation in which to run the instance
+   * @default - No specific Capacity Reservation ID
+   */
+  readonly capacityReservationId?: string;
+
+  /**
+   * The ARN of the Capacity Reservation resource group in which to run the instance
+   * @default - No Capacity Reservation resource group
+   */
+  readonly capacityReservationResourceGroupArn?: string;
+}
+
+/**
+ * Capacity Reservation specification for LaunchTemplate
+ * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-launchtemplate-launchtemplatedata-capacityreservationspecification.html
+ */
+export interface LaunchTemplateCapacityReservationSpecification {
+  /**
+   * Indicates the instance's Capacity Reservation preferences
+   * @default - No preference specified
+   */
+  readonly capacityReservationPreference?: CapacityReservationPreference;
+
+  /**
+   * Information about the target Capacity Reservation or Capacity Reservation group
+   * @default - No target specified
+   */
+  readonly capacityReservationTarget?: LaunchTemplateCapacityReservationTarget;
```

Comment:
Shouldn't `target` be required if you specify a capacity reservation?

I can now specify: 

```ts
capacityReservationSpecification: { }
```

What does that mean?